src/BlockBandedMatrices.jl

```diff
@@ -45,7 +45,7 @@ import BandedMatrices: isbanded, bandwidths, bandwidth, banded_getindex, colrang
                         _banded_colval, _banded_rowval, _banded_nzval # for sparse
 
 export BandedBlockBandedMatrix, BlockBandedMatrix, BlockSkylineMatrix, blockbandwidth, blockbandwidths,
-        subblockbandwidth, subblockbandwidths, Ones, Zeros, Fill, Block, BlockTridiagonal, BlockBidiagonal, isblockbanded
+        subblockbandwidth, subblockbandwidths, Ones, Zeros, Fill, Block, BlockDiagonal, BlockTridiagonal, BlockBidiagonal, isblockbanded
```

Comment:
Do we want to export `BlockDiagonal`? It will make this package incompatible with https://github.com/invenia/BlockDiagonals.jl

Reply:
I don't need it myself at the moment, I just added the export to be consistent. One could argue that BlockBandedMatrices.jl is the better place for `BlockDiagonal`, but I have no strong opinion.

Reply:
If we don’t export it we can do a patch version, right? (That is, there are no breaking changes in this PR) if so maybe we don’t export for now

Reply:
Fair enough. I've bumped the version number.